### PR TITLE
prepare release 0.13.52

### PR DESCRIPTION
**Pull request by nickchapman-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/release-notes.rst [hunk 1]
```diff
@@ -6,6 +6,91 @@ Release notes
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- The time mode must now always be
+  specified explicitly. Use ``--static-time`` to recover the previous
+  default time mode.
+
+JSON API - Experimental
+~~~~~~~~~~~~~~~~~~~~~~~
+- wrap Streaming API events in JSON object:
+   ``{ "events": [ E1, E2, ... ] }``
+   See: 4384
```
Comment (hurryabit) [inline docs/source/support/release-notes.rst]:
```suggestion
   See `issue #4384 <https://github.com/digital-asset/daml/issues/4384>`_.
```

File: docs/source/support/release-notes.rst [hunk 2]
```diff
@@ -6,6 +6,91 @@ Release notes
...
+
+- The format of ``archived`` responses from WebSocket endpoints
+  has changed to include template IDs, similar to exercise responses.
+  See `issue #4383 <https://github.com/digital-asset/daml/issues/4383>`_.
+
+- Rename JSON API endpoints. See #4289 and #3145
```
Comment (hurryabit) [inline docs/source/support/release-notes.rst]: Please use the same link format as above.

File: docs/source/support/release-notes.rst [hunk 3]
```diff
@@ -6,6 +6,91 @@ Release notes
...
+    /contracts/lookup => /v1/fetch
+    /contracts/search => /v1/query
+    /contracts/searchForever => /v1/stream/query
+    /contracts/lookupForever => /v1/stream/fetch
+    /parties => /v1/parties
+
+- Exercise response field "contracts" renamed to "events".
+  See `issue #4385 <https://github.com/digital-asset/daml/issues/4385>`_.
+
+- Added streaming version of fetch by key: ``/stream/fetch``. See #4075.
```
Comment (hurryabit) [inline docs/source/support/release-notes.rst]: Link format once more.

File: docs/source/support/release-notes.rst (hunk 1 above)
Comment (nickchapman-da) [reply docs/source/support/release-notes.rst]: done

**Comment by nickchapman-da** [comment]
All manual tests have passed. I tested Linux. @cocreature tested Windows.

**Pull request merged**